app/src/main/res/values/strings.xml

```diff
@@ -684,5 +684,15 @@
     <string name="sell_warning_only_sol_body">To Cash out another coin swap your cryptocurrency to SOL before</string>
     <string name="sell_warning_only_sol_button">Got it</string>
 
+    <string name="sell_inform_dialog_next_step">service is next step</string>
+    <string name="sell_inform_dialog_first_step">1</string>
+    <string name="sell_inform_dialog_first_title">You will be redirected to our payment provider</string>
+    <string name="sell_inform_dialog_first_subtitle">You will need to enter your IBAN and pass KYC</string>
+    <string name="sell_inform_dialog_second_step">2</string>
+    <string name="sell_inform_dialog_second_title">Transfer SOL to our payment provider from Key App</string>
+    <string name="sell_inform_dialog_second_subtitle">Nobody has access to your funds, so you need to execute the transaction cash out</string>
+    <string name="sell_inform_dialog_do_not_show">Don’t show me again</string>
```

Comment:
don't forget to escape '  with \'

Reply:
👍
yes we need to replace Don’t on Don/'t